### ISSUE TITLE
Consider dropping 1.9.3 support for next release?

### ISSUE THREAD

**Issue by jgeewax** [GoogleCloudPlatform/gcloud-ruby#432]
We're coming up on the 9-month mark for EOL of 1.9.3 and it won't be receiving security bug fixes anymore ! (https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/)

Also, gRPC requires 2.0+ (as it needs keyword args), so if we want to ever use gRPC, we'll need to drop 1.9.3 support. (https://github.com/grpc/grpc/issues/4020)

What are your thoughts on dropping 1.9.3 support ?

**Comment by blowmage** [comment]
We are in favor of it. The next release of rails will require 2.2, which I think will influence lots of gems.

Just this week we had a gem that gcloud has an indirect dependency on release a new version that dropped 1.9 support to take advantage of  ruby 2 syntax and features. So we had to add a dependency on an earlier version to keep the 1.9 build green. I expect we will see more of this in the future.

**Comment by jgeewax** [comment]
Awesome.

@quartzmo Do you agree?

If so, let's make this more firm and say "before the end of 2015, we're going to require Ruby 2.0+" ?

**Comment by jgeewax** [comment]
This also unblocks us with gRPC for Linux, so I'll open another issue on what we should do for that. For now, installing on Ubuntu 15.10 actually works: https://github.com/grpc/grpc/issues/4111

**Comment by quartzmo** [comment]
@jgeewax I agree! But I think it's good procedure to leave this issue open for comments for the timeline you suggest.

**Comment by jgeewax** [comment]
SGTM :)
